tests/backdrop/StateCommandsTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * @file
+ * PHPUnit tests for Bee State commands.
+ */
+
+use PHPUnit\Framework\TestCase;
+
+class StateCommandsTest extends TestCase {
+
+  /**
+   * Make sure that the state-get command works.
+   */
+  public function test_state_get_command_works() {
+    // New installs don't have maintenance_mode state so need setting to TRUE to initiate
+    $output = shell_exec('bee maintenance-mode TRUE');
+    $output = shell_exec('bee maintenance-mode FALSE');
+
+    // Make sure maintenance mode is disabled and message matches format
+    $output = shell_exec('bee state-get maintenance_mode');
+    $this->assertStringContainsString("The value (boolean) of the 'maintenance_mode' state is: FALSE", (string) $output);
+  }
+
+  /**
+   * Make sure that the state-set command works.
+   */
+  public function test_state_set_command_works() {
+    // Make sure maintenance_mode is disabled.
+    $output = shell_exec('bee state-get maintenance_mode');
+    $this->assertStringContainsString("The value (boolean) of the 'maintenance_mode' state is: FALSE", (string) $output);
+
+    // Set maintenance_mode to 'TRUE'.
+    $output = shell_exec('bee state-set maintenance_mode TRUE');
+    $this->assertStringContainsString("The 'maintenance_mode' state was set to: TRUE", (string) $output);
+
+    // Make sure maintenance_mode is enabled.
+    $output = shell_exec('bee state-get maintenance_mode');
+    $this->assertStringContainsString("The value (boolean) of the 'maintenance_mode' state is: TRUE", (string) $output);
+  }
+
+  /**
+   * Make sure that the maintenance-mode command works.
+   */
+  public function test_maintenance_mode_command_works() {
+    // Attempt to enable maintenance mode
+    $output = shell_exec('bee maintenance-mode TRUE');
+    $this->assertStringContainsString('Maintenance mode is already enabled', (string) $output);
+
+    // Disable maintenance mode
+    $output = shell_exec('bee maintenance-mode FALSE');
+    $this->assertStringContainsString('Maintenance mode was disabled', (string) $output);
+
+    // Enable maintenance mode
+    $output = shell_exec('bee maintenance-mode TRUE');
+    $this->assertStringContainsString('Maintenance mode was enabled', (string) $output);
+
+    // Get the status of maintenance mode
+    $output = shell_exec('bee maintenance-mode');
+    $this->assertStringContainsString('Maintenance mode is enabled', (string) $output);
+
+    // Disable maintenance mode
+    $output = shell_exec('bee maintenance-mode FALSE');
+    $this->assertStringContainsString('Maintenance mode was disabled', (string) $output);
+
+    // Get the status of maintenance mode
```

Comment:
```suggestion
    // Get the status of maintenance mode.
```

Reply:
@klonos this whole thing would end up in 33 commits. Each of the commits would trigger a test run. Maybe a "real" PR against the existing one makes more sense? And it would be way easier to read (and review). What do you think?

Reply:
I was going to work through manually as need to update tests so should be one